tools/pythonpkg/scripts/sqllogictest_python.py

```diff
@@ -168,10 +172,15 @@ def main():
             continue
         if test_directory:
             file_path = os.path.join(test_directory, file_path)
-        test = sql_parser.parse(file_path)
-        if not test:
-            print(f'Failed to parse {file_path}')
-            exit(1)
+
+        try:
+            test = sql_parser.parse(file_path)
+        except SQLParserException as e:
+            if not ("test" in locals()):  # test hasn't been initialized because of the raised exception
+                test = SQLLogicTest("")
+            test.skip(True)
+            executor.skip_log.append(str(e.message))
```

Comment:
I think we can `continue` after this
Then we don't need to add `skip` to `SQLLogicTest`, so we don't need a `skipped` variable in there

Reply:
True! Thanks!